docs/versioned_docs/version-3.5/technical-reference/contributing.md

```diff
@@ -57,6 +57,10 @@ This describes the overall steps to your first code contribution in OpenRefine.
 
 - Study how the current code works. You might want to use a debugger to put breakpoints at the relevant locations (for inspecting the backend, use your IDE's debugger, for the frontend, use your browser's developer tools).
 
+- Before going too far in the exploration and solving of the issue, make sure no one is already assigned to it. If someone is already assigned, and you don't see action or discussions since a few weeks, you can ask the assigned person if he is still currently active in working to solve the issue. If not, or no answer after a few days, you can remove him and assign yourself.
+
+- Please note that in periods when the OpenRefine community is actively participating in a program like Outreachy or Google Summer of Code, it might be safer to ask mentors before assigning yourself to « Good First Issue » or « Good Second Issue ».
```

Comment:
```suggestion
- Please note that in periods when the OpenRefine community is actively participating in a program like Outreachy or Google Summer of Code, it might be safer to ask mentors before assigning yourself to a "good first issue" or "good second issue".
```